INSTRUCTIONS.md

```diff
@@ -0,0 +1,83 @@
+# Rick and Morty
+​
+## Assignment
+​
+Build an application that displays the characters’ profiles of the Rick & Morty sitcom using the [Rick & Morty API](https://rickandmortyapi.com/documentation/).
+​
+The app should be a React application built with TypeScript.
+​
+You’re free to choose whether to implement standard pagination or infinite scrolling.
+​
+You’re free to choose whether to use the REST API or GraphQL API.
+​
+You can find the design we expect you to implement [here](https://www.figma.com/file/FXostZn4wrjuNr5W4D5PGA/Rick-and-Morty-(web-responsive)-(Community)?node-id=97%3A196&amp;amp;t=HV2hIQo7VrgnlR1f-1).
+​
+The project has already the following scripts setup:
+
+- android: Starts the Android app
+- ios: Starts the iOS app
+- test: Run tests using Jest
+- typecheck: Typecheck code with TypeScript
+- start: Starts the Metro bundler
+​
+Styling the app with a CSS-in-JS solution, like styled-components, is a plus. You should not use ready-to-use design system libraries (e.g., Chakra UI, Material UI, Bootstrap, etc.).
+​
+You’re free to pick the vendors you consider necessary for the exercise, but be careful that overcomplicating the implementation will lead to a bad evaluation.
+
+You're free to choose the platform (iOS or Android) on which you want to develop the application. Be careful that this platform will be the one on which we'll evaluate your assignment.
+
+### Heads UP
+​
+The project shouldn’t rely on global dependencies. The coding challenge will be considered unsuccessful if the project won’t compile after we install its dependencies.
+​
+If a particular version of Yarn should be used to run the project, it should be specified in the README.
```

Comment:
I think that we should have a local version of Yarn v3 in the project, just like we have in the monorepo.

Reply:
This would remove the burden on the candidate. I agree.